<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(96, 181, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="productBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:20px;
color: rgb(58, 141, 255);
font: 75 10pt &quot;Eras Bold ITC&quot;;</string>
    </property>
    <property name="text">
     <string>Products</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salesreportBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>280</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:20px;
color: rgb(58, 141, 255);
font: 75 10pt &quot;Eras Bold ITC&quot;;</string>
    </property>
    <property name="text">
     <string>Sales Report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="makeorderBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>230</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:20px;
color: rgb(58, 141, 255);
font: 75 10pt &quot;Eras Bold ITC&quot;;</string>
    </property>
    <property name="text">
     <string>Make Order</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 236, 219);
font: 75 12pt &quot;Eras Bold ITC&quot;;</string>
    </property>
    <property name="text">
     <string>WELCOME TO YOUR DAILY SALES TRACKER</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="choices">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 141, 255);
color:rgb(251, 243, 242);
font: 75 10pt &quot;Eras Bold ITC&quot;;
border: 0.5px solid #1C768F;
border-radius: 8px;
padding-left:11px;</string>
    </property>
    <property name="currentText">
     <string>Dashboard</string>
    </property>
    <item>
     <property name="text">
      <string>Dashboard</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Account</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
font: 75 10pt &quot;Verdana&quot;;
font-weight:350;

</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Using the DailySales tracker, you can add products, make orders, and track your sales history daily.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../product.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dateTimeLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 243, 242);;
color:rgb(3, 37, 57);
font: 75 10pt &quot;Verdana&quot;;
border: 1px solid rgb(28, 118, 143);
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="monthlyOrdergraphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>521</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 5pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="graphorderwidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>521</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 5pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
